test/SlowTests/Issues/RavenDB-15330.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Server.Replication;
+using Orders;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Util;
+using Raven.Server.Config;
+using Raven.Server.Rachis;
+using Raven.Server.ServerWide;
+using Raven.Server.ServerWide.Commands.Indexes;
+using Raven.Server.ServerWide.Commands.Subscriptions;
+using Raven.Server.ServerWide.Context;
+using Raven.Server.Utils;
+using SlowTests.Rolling;
+using Sparrow.Json.Parsing;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+public class RavenDB_15330 : ClusterTestBase
+{
+    public RavenDB_15330(ITestOutputHelper output) : base(output)
+    {
+    }
+
+    [Fact]
+    public async Task Command_With_DontCareId_Should_Be_Committed()
+    {
+        DebuggerAttachedTimeout.DisableLongTimespan = true;
+        var server = GetNewServer();
```

Comment:
let's use `DoNotReuseServer` method instead

Reply:
Done

Reply:
if you have `DoNotReuseServer`, do you still need this server?